teku/src/main/java/tech/pegasys/teku/cli/options/P2POptions.java

```diff
@@ -158,6 +158,46 @@ public class P2POptions {
       arity = "1")
   private boolean multiPeerSyncEnabled = SyncConfig.DEFAULT_MULTI_PEER_SYNC_ENABLED;
 
+  @Option(
+      names = {"--Xp2p-historical-sync-batch-size"},
+      paramLabel = "<NUMBER>",
+      showDefaultValue = Visibility.ALWAYS,
+      description =
+          "Number of objects being requested in a single batch to a single peer, while forward syncing",
+      hidden = true,
+      arity = "1")
+  private Integer historicalSyncBatchSize = SyncConfig.DEFAULT_HISTORICAL_SYNC_BATCH_SIZE;
+
+  @Option(
+      names = {"--Xp2p-sync-batch-size"},
+      paramLabel = "<NUMBER>",
+      showDefaultValue = Visibility.ALWAYS,
+      description =
+          "Number of objects being requested in a single batch to a single peer, while forward syncing",
```

Comment:
I am kind of not a fan of `Number of objects being requested` . Find it confusing. Don't have better suggestion. Does `blocks/blobs` makes more sense?

Reply:
I put this PR to actually start the conversation on this :)
I blocks\blobs makes more sense but then should be mention the maxBlobsPerBlock multiplier?

Reply:
Yeah, I think it's fine to merge as is since they are hidden options